crates/dkg-cli/Cargo.toml

```diff
@@ -19,11 +25,15 @@ hex = "0.4.2"
 anyhow = "1.0.31"
 
 ethers = { git = "https://github.com/gakonst/ethers-rs", features = ["celo"] }
+ethers-solc = { git = "https://github.com/gakonst/ethers-rs" }
```

Comment:
Can you pin all git dependencies to commits so the build won't break in the future? See https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories

Reply:
Pinned ethers-rs to 0.17.0